### DISCUSSION TITLE
How to get customized list of data without relation

### DISCUSSION THREAD

**Discussion by sreelal-cn** [rebing/graphql-laravel#836] [Q&A]
I fetch data from users, I have not direct relation user and topic.

I just, assigned topic data in the resolve part as, 
$user->topics = $topicData;

```
'topics' => [
                'type'          => Type::listOf(GraphQL::type('AssignTopic')),
                'description'   => 'The topics',
                'is_relation'   => false,
                'selectable'    => false,
                'default'       => null
            ],
```

When I hitting the query, It still wants a relationship. 

How to bypass the relation?

`"debugMessage": "Call to undefined relationship [topics] on model [App\\Models\\User].",`

**Comment by mfn** [comment]
Might be actually the default resolver, which AFAIK would do that.

You talk about your resolver, but you didn't show it: can you please provide more complete examples?
